src/Microsoft.ML.Transforms/Text/WordBagTransform.cs

```diff
@@ -451,8 +451,8 @@ public sealed class TermLoaderArguments
         [Argument(ArgumentType.AtMostOnce, IsInputFileName = true, HelpText = "Data file containing the terms", ShortName = "data", SortOrder = 2, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly)]
         public string DataFile;
 
-        [Argument(ArgumentType.Multiple, HelpText = "Data loader", NullName = "<Auto>", SortOrder = 3, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly)]
-        public SubComponent<IDataLoader, SignatureDataLoader> Loader;
+        [Argument(ArgumentType.Multiple, HelpText = "Data loader", NullName = "<Auto>", SortOrder = 3, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, SignatureType = typeof(SignatureDataLoader))]
```

Comment:
>Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly [](start = 103, length = 57)

unrelated to your PR, but we need to look at some point on this visibility. It's CmdLineOnly now because Subcomponent can't be exposed to EntryPoints, but since you moving to IComponentFactory, maybe we should remove this visibility restriction.